k8s/apis/solution/v1/solution_webhook.go

```diff
@@ -126,7 +128,7 @@ func (r *Solution) validateCreateSolution() error {
 
 func (r *Solution) validateUpdateSolution() error {
 	var solutions SolutionList
-	err := mySolutionClient.List(context.Background(), &solutions, client.InNamespace(r.Namespace), client.MatchingFields{".spec.displayName": r.Spec.DisplayName})
+	err := mySolutionClient.List(context.Background(), &solutions, client.InNamespace(r.Namespace), client.MatchingFields{".spec.displayName": r.Spec.DisplayName}, client.Limit(2))
```

Comment:
why limit is 2 here?

Reply:
this is an update check, the result may contain itself